src/udev/udev-ctrl.c

```diff
@@ -340,6 +341,13 @@ int udev_ctrl_send(UdevCtrl *uctrl, UdevCtrlMessageType type, const void *data)
         return 0;
 }
 
+int udev_ctrl_send_ping(UdevCtrl *uctrl) {
+        assert(uctrl);
+        assert(uctrl->varlink);
+
+        return udev_varlink_call(uctrl->varlink, "io.systemd.udev.Ping", NULL, NULL);
```

Comment:
i think this concept is so generic that we should make it generic, i.e. io.systemd.service.Ping() or so that we can implement in many services generically.